### ISSUE TITLE
is this project still active?

### ISSUE THREAD

**Issue by laoshaw** [visionmedia/page.js#611]
Looking for a SPA router for svelte and found page.js has no updates for two years, is this still an active project?

**Comment by Clemens-B** [comment]
@laoshaw same here.. Did you find one for svelte SPAs?

**Comment by laoshaw** [comment]
svelte-spa-router is the one

**Comment by githraj** [comment]
> Looking for a SPA router for svelte and found page.js has no updates for two years, is this still an active project?
I don't know as well.

**Comment by oskarrough** [comment]
Last commit was over three years ago. Check the amount of issues reported and pull requests never merged. It's obviously not an active project any more.

BUT, this is the important part. It works just fine as is in my experience.

**Comment by 4www** [comment]
Also been using this router for every project in the last few years, and haven't been encountering a bug. I'd say it is a finished, and trustworthy software in the javascript ecosystem.

**Comment by vinyll** [comment]
Same here: I've been using this in production since 2018 and it's all been perfect.
I guess we are not used that a project can just be working and completed.
I consider this project as just stable and long-term.

**Comment by matthewp** [comment]
Is anyone willing to maintain it?

**Comment by klauss194** [comment]
Worth trying

**Comment by carlosmintfan** [comment]
@matthewp are you the same one as @matthewpblog, of whom the last commits are? Although the repo actually belongs to @visionmedia?? I‘d like to maintain. Would you give me write access or should I fork?

**Comment by vinyll** [comment]
Heyh @carlosmintfan, that's great that you want to maintain!
You may probably want to show you're totally able to take over by forking and providing with a couple or pull requests; that will show how well you understand the core and improve.
Looking forward for a new kick ;)

**Comment by carlosmintfan** [comment]
Thanks @vinyll! I‘ll start working in about a week, when I have more time for programming. The most important thing to be done at first would be to bump path-to-regexp (see #616) because currently you have to choose between latest version of page-js or vulnerable version of path-to-regexp.

**Comment by matthewp** [comment]
Yes, I'm the same Matthew. To avoid handing over maintenance to a bad actor want to make sure anyone interested in becoming a maintainer has a track record in the open source world.

**Comment by carlosmintfan** [comment]
@matthewp How are you applying that to me? I have some merged PRs...
Anyway, I'm not that fast here, but I've done a little bit of stuff two weeks ago. It may take some time until I actually publish something. Thank you for your patience.

**Comment by carlosmintfan** [comment]
I've published a fork named `page-two` at npm: https://www.npmjs.com/package/page-two.
The problem is there's no `@types/page-two` yet. But I'll switch the code to TypeScript itself later.

**Comment by carlosmintfan** [comment]
I have unpublished the package from npm due to me not having done necessary changes after bumping to a new path-to-regexp version. While it seemed to work, I don’t want you to experience bugs so I‘ll publish this as soon as I have the new version ready. I‘m working on making it TypeScript, there I saw I have to make a change because of the new path-to-regexp version.

**Comment by carlosmintfan** [comment]
Hi all 👋,

Sorry for the delay. This is not my top priority at the moment, but I've finally published page-two version 1.1.0 now. It uses TypeScript. Executing tests doesn't work yet, but page-two works for me. Please test yourself LOL. I'll fix running the tests at some point in the future.
